### ISSUE TITLE
The problem of training

### ISSUE THREAD

**Issue by DragonZzzz** [BinWang-shu/pytorch_hed#3]
When I train BSDS500 dataset, the images in tmp are black, and the loss is almost zero. The training has crashed.

**Comment by BinWang-shu** [comment]
Maybe the reason is on the label. In this repositories, I have pre-processed the label. The label is binary. You also can revise the dataset.py to add pre-process.

**Comment by MillenniumFalcon1097** [comment]
i meet the same problem. Could you provide with the pre-process code?

**Comment by MillenniumFalcon1097** [comment]
> Maybe the reason is on the label. In this repositories, I have pre-processed the label. The label is binary. You also can revise the dataset.py to add pre-process.

I have already generated the gt label and saved as png, however after I add the labelPath in the function __init__ and __getitem__ of the class BSDS500, when training it still appears very big loss and black images in /tmp.
